src/features/courses/course-repository.ts

```diff
@@ -532,9 +536,28 @@ class CourseRepository {
         }
     }
 
+    async createStudentGradeOverride(obj: Partial<StudentGradeOverride>): Promise<StudentGradeOverride> {
+        try {
+            return await StudentGradeOverride.create(obj);
+        } catch (e) {
+            // Check constraint errors if there are any
```

Comment:
Does the Wrapped error check constraints itself?

Reply:
Nope, all it does is combine the stack of the error with the current stack
sequelize errors stacks never point back to our code which is why this is so helpful
This create doesn't take any user input that should break it, therefore if there is an issue it's an internal server error